_delphi_utils_python/delphi_utils/runner.py

```diff
@@ -77,6 +78,10 @@ def run_indicator_pipeline(indicator_fn:  Callable[[Params], None],
     else:
         t1.terminate()
         t1.join()
+    elapsed_time_in_seconds = round(time.time() - start_time, 2)
+    logger.info("Completed flash step",
```

Comment:
how about making this conditional on normal flash completion, with a different error-level log message when the flash process is forcefully terminated?